app/assets/javascripts/users.js

```diff
@@ -65,6 +65,11 @@ WebsiteOne.define('Users', function () {
         });
 
         $('#require_users_profile').modal({show: true});
+
+        // Enable bootstrap tooltips on non-touch devices only
+        if (!('ontouchstart' in document.documentElement)) {
+            $('[data-toggle="tooltip"]').tooltip({delay: { "show": 500, "hide": 100 }});
```

Comment:
@tansaku - You might see tooltips but do you see bootstrap tooltips?  That is the primary function of this bit of code, to enable bootstrap tooltips - it passes in options to control the delay between showing and hiding them, but it is necessary to call this function according to the bootstrap documentation: http://getbootstrap.com/javascript/#four-directions - look at the "opt in functionality" bit.
The comment was more about the why it is doing this than what it is doing but ok, I could refactor to a function called
`initialiseBootstrapsTooltipsForNonTouchDevices`, or would you prefer `initialiseBootstrapTooltipsForNonTouchDevicesWithDelayBeforeShowing`?
The touch related js is all over the shop - stack overflow, blogs, github... https://www.google.co.uk/search?q=%27ontouchstart%27+in+document.documentElement